packages/PriceLockup/__tests__/PriceLockup.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import PriceLockup from '../PriceLockup'
+
+describe('PriceLockup', () => {
+  const doShallow = (props = {}) => shallow(<PriceLockup {...props} />)
+
+  it('renders', () => {
+    const priceLockup = doShallow()
+
+    expect(priceLockup).toMatchSnapshot()
+  })
+
+  it('does other things', () => {
```

Comment:
A bit of a vague description for this test. Can this be improved, or if it's unnecessary be removed?

Reply:
This can be removed.

Reply:
done